app/main.py

```diff
@@ -1,6 +1,16 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    unique_values = {}
+
+    @wraps(func)
+    def inner(*args) -> Any:
```

Comment:
Please enhance this make it work with `**kwargs` as well

Reply:
Hello, I have added an opportunity to work with *kwargs. BUT for what if README file says that "Decorator cache creating for decorating only functions that take **immutable arguments.**"

Reply:
Hello, I've added an opportunity to work with kwargs